src/test/coins_tests.cpp

```diff
@@ -142,6 +142,20 @@ class CCoinsViewTest : public CCoinsView
         mapNullifiers.clear();
     }
 
+    template<typename Tree, typename Cache>
+    void BatchWriteAnchors(Cache& mapAnchors, std::map<uint256, Tree>& cacheAnchors)
```

Comment:
Why is `mapAnchors` a `Cache`, and `cacheAnchors` a `std::map`?

Reply:
This does seem counterintuitive. This is based on how it is done in coins.cpp
```
template<typename Map, typename MapIterator, typename MapEntry>
void BatchWriteAnchors(
    Map &mapAnchors,
    Map &cacheAnchors,
    size_t &cachedCoinsUsage
)
```
and also 
```
void BatchWriteNullifiers(CNullifiersMap& mapNullifiers, std::map<uint256, bool>& cacheNullifiers)
``` 
in coins_tests.cpp

I think I called it Cache because the field we pass in, ```mapSproutAnchors_``` in ```CCoinsViewTest```, is equivalent to the field ```cacheSproutAnchors``` in ```CCoinsViewCache```

I could changed it from ```typename Cache``` to ```typename Map```, but I'm neutral. What do you think?

Reply:
`mapSproutAnchors_` is passed in as the second argument `cacheAnchors`. The first argument `mapAnchors` is `mapSproutAnchors`, which AFAICT is the map. So yes, please change it to `typename Map`.